config/application.rb

```diff
@@ -35,6 +35,10 @@ class Application < Rails::Application
     # Only loads a smaller set of middleware suitable for API only apps.
     # Middleware like session, flash, cookies can be added back manually.
     # Skip views, helpers and assets when generating a new resource.
-    config.api_only = true
+    config.middleware.insert_before 0, Rack::Cors do
+      allow do
+        origins Rails.application.credentials.allowed_origins
+      end
+    end
```

Comment:
Although stackoverflow can be handy but i would refrain from using them much. Since Rails 5.2, we have a new way of handling the credentials in rails. Check this out: https://guides.rubyonrails.org/security.html#custom-credentials. 

You can use the credentials.yml.enc file to store your credentials for s3 as well as mention your `allowed_origins` in the file. be careful with the master.key file.

Reply:
@rahulbajaj0509  ok. I will into it. Can you please review and merge the PRs for locales and versions API so that we have only screenshots API remaining for next meeting discussion.